CONTRIBUTING.md

```diff
@@ -128,32 +131,17 @@ Meilisearch tools follow the [Semantic Versioning Convention](https://semver.org
 This project integrates a bot that helps us manage pull requests merging.<br>
 _[Read more about this](https://github.com/meilisearch/integration-guides/blob/main/resources/bors.md)._
 
-### Automated Changelogs <!-- omit in TOC -->
-
-This project integrates a tool to create automated changelogs.<br>
-_[Read more about this](https://github.com/meilisearch/integration-guides/blob/main/resources/release-drafter.md)._
+### Release on Github and npm
 
-### How to Publish the Release <!-- omit in TOC -->
+This repository uses the [changesets](https://github.com/Noviny/changesets) library to handle the version updates and the publishing on Github and `npm`.
 
-⚠️ Before doing anything, make sure you got through the guide about [Releasing an Integration](https://github.com/meilisearch/integration-guides/blob/main/resources/integration-release.md).
+When PR's are merged on `main`, they trigger the [`release` CI](./.github/workflows/publish.yml). This CI creates a PR titled `Version Packages` containing all the version updates and changelogs of the impacted packages. The PR updates the versions and the changelogs based on the `changesets` that were previously pushed on main (see [changeset section](#changesets)).
 
-#### Version update
-Make a PR modifying the following files with the right version:
+To release on Github and `npm` you must merge the `Version packages` PR. This will trigger the publishing action and create the Github and `npm` releases for all affected packages.
 
-[`package.json`](/package.json):
-```javascript
-"version": "X.X.X",
-```
-
-[`src/package-version`](/src/package-version.ts)
-```javascript
-export const PACKAGE_VERSION = 'X.X.X'
-```
+See more in depth explaination on [versioning](https://github.com/changesets/changesets/blob/main/docs/command-line-options.md#version), [publishing](https://github.com/changesets/changesets/blob/main/docs/command-line-options.md#publish) and the [changesets github-action](https://github.com/changesets/action).
 
-#### Github publish
-Once the changes are merged on `main`, you can publish the current draft release via the [GitHub interface](https://github.com/meilisearch/instant-meilisearch/releases): on this page, click on `Edit` (related to the draft release) > update the description (be sure you apply [these recommendations](https://github.com/meilisearch/integration-guides/blob/main/resources/integration-release.md#writting-the-release-description)) > when you are ready, click on `Publish release`.
-
-GitHub Actions will be triggered and push the package to [npm](https://www.npmjs.com/package/@meilisearch/instant-meilisearch).
+If you were previously on `pre-release mode`, do not forget to create a PR exiting the pre-release mode by doing `yarn changeset pre exit` and then merge it to main.
```

Comment:
pre-release mode?
I don't understand what it means

Reply:
Suggestion: 

If you merged a beta branch into main, that you now want to release, you were probably in the `changesets` [pre-release](https://github.com/changesets/changesets/blob/main/docs/prereleases.md) mode (see section on [releasing a beta](#release-a-beta-version)). If the `pre.json` file is present in the `.changesets` folder, you need to exit that mode. This is possible by running `yarn changeset pre exit`. Once done, create a PR with the changes and merge it to main.